.dockerignore

```diff
@@ -0,0 +1,8 @@
+kube
```

Comment:
I think that is fair that a `kube` folder not be in an open source project, and probably it is too specific a name. I would like to use a deployment folder in some services SO are redeveloping with the help of DSP. When we use a deployment folder, I want it to be ignored from docker.
Conclusion, are you fine with having a "deployment" folder referred to in the .dockerignore file?